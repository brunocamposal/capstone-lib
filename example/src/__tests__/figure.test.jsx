import React from 'react'
import renderer from 'react-test-renderer'
import { Figure } from 'capstone-lib/'

describe('Figure: ', () => {
  it('renders the component', () => {
    const tree = renderer
      .create(<Figure alt='coisa' url='http://teste.coisa.com' y={140} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
