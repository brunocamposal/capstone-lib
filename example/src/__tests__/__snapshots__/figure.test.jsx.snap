// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Figure:  renders the component 1`] = `
<figure
  className="sc-gtssRu eTYvhi"
>
  <img
    alt="coisa"
    className="sc-dlnjPT iAgkHO"
    style={
      Object {
        "maxHeight": "140px",
        "maxWidth": "100%",
      }
    }
  />
  <figcaption
    className="sc-hKFyIo jjUcTb"
  >
    “
    coisa
    ”
  </figcaption>
</figure>
`;
